cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,35 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
-  });
-
-  it('', () => {
+describe('Conduit', () => {
+  const userName = 'nevermore';
+  const email = 'nevermore@gmail.com';
+  const password = 'nevermore123';
+  const title = 'title';
+  const description = 'description';
+  const body = 'body';
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.login(email, userName, password);
   });
 
-  it('', () => {
+  it('should create article with valid data', () => {
+    cy.reload();
```

Comment:
The `reload` function is called at the start of the test. This might not be necessary as the `beforeEach` hook already ensures that the page is visited before each test.